pom.xml

```diff
@@ -55,6 +55,27 @@
         <logging.logLevel>debug</logging.logLevel>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-annotations</artifactId>
+                <version>2.14.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>2.0.0</version>
+            </dependency>
```

Comment:
I think `snakeyaml` should be added here too

Reply:
Can you bump version to remove CVE alerts besides ?https://github.com/tarantool/cartridge-java-testcontainers/issues/58

```
Dependency org.yaml:snakeyaml:1.26 is vulnerable
CVE-2022-25857 7.5 Uncontrolled Resource Consumption vulnerability pending CVSS allocation
CVE-2022-38752 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38749 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38750 5.5 Out-of-bounds Write vulnerability pending CVSS allocation
CVE-2022-38751 6.5 Out-of-bounds Write vulnerability pending CVSS allocation
Results powered by Checkmarx(c)
```

Reply:
Hm, let's try to use 1.33

Reply:
Done